### PR TITLE
Platform — Fixed declarations

### PR DESCRIPTION
**Pull request by monolithed**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [c] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

The changes:

* Added `declare module 'platform'` for resolve the module;
* Internal interface `PlatformStatic` was replaced with Platform
* Internal`PlatformOs` was merged with Platform
* TSLint errors were fixed

### CODE REVIEW

File: platform/index.d.ts [hunk 1]
```diff
@@ -13,18 +12,19 @@ interface PlatformStatic {
...
+        architecture?: number;
+        family?: string;
+        version?: string;
+        toString(): string;
+    };
+    parse?(ua: string): Platform;
     toString?(): string;
 }
 
-interface PlatformOS {
```
Comment (yuit) [inline platform/index.d.ts]: Why you remove this?

Comment (monolithed) [inline platform/index.d.ts]: PlatformOS is a private interface and does not any sense to export.

File: platform/index.d.ts [hunk 2]
```diff
@@ -1,10 +1,9 @@
-// Type definitions for Platform 1.0.0
+// Type definitions for Platform 1.3
 // Project: https://github.com/bestiejs/platform.js
 // Definitions by: Jake Hickman <https://github.com/JakeH/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-
-interface PlatformStatic {
+declare interface Platform {
```
Comment (yuit) [inline platform/index.d.ts]: nit: no need for the declare here

Comment (monolithed) [inline platform/index.d.ts]: What do you mean. Here is a working example:
img-2017-03-21-10-31-26

Comment (yuit) [inline platform/index.d.ts]: So interface is only used as type so `declare interface` or `interface` are the same.... both of them work

**Pull request merged**